src/odemis/acq/stream/_projection.py

```diff
@@ -436,3 +530,617 @@ def _updateImage(self):
 
         except Exception:
             logging.exception("Updating %s %s image", self.__class__.__name__, self.name.value)
+
+
+class MeanSpectrumProjection(DataProjection):
```

Comment:
Please sort the classes in sort-of hierarchical order. I'd suggest to leave the RGBProjections at the top. All these extra "1D" DataProjection should be at the end of the file.